### ISSUE TITLE
Not working on macOS 13 beta

### ISSUE THREAD

**Issue by UptonEdward** [Eugeny/tabby#6564]
**Describe the problem**:
I'm using the latest version of Tabby and it's not working on macOS 13 beta. There is no error message, the GUI just won't show up despite the app is in fact running.

**To Reproduce**:
Just upgrade macOS to Ventura and open Tabby.

**Comment by kshq1996** [comment]
what`s chip

**Comment by JaheimLee** [comment]
Same problem！My chip is M1

**Comment by UptonEdward** [comment]
> what`s chip

Intel i5

**Comment by xinkunZ** [comment]
same here,  M1 macbook air

**Comment by opteroncx** [comment]
same, M1

**Comment by ZoeyCluff** [comment]
M1 pro reporting in
![CleanShot 2022-06-08 at 12 16 36](https://user-images.githubusercontent.com/2716914/172677612-317204cb-932e-4c7d-bfd0-6fec7f3b99d5.png)
![CleanShot 2022-06-08 at 12 16 43](https://user-images.githubusercontent.com/2716914/172677639-9ec24ce8-9235-4cc4-b343-37b444405e83.png)

**Comment by piersandro** [comment]
Launching from MacOS Terminal shows this error:
```
❯ ./Tabby
objc[4587]: Class WebSwapCGLLayer is implemented in both /System/Library/Frameworks/WebKit.framework/Versions/A/Frameworks/WebCore.framework/Versions/A/Frameworks/libANGLE-shared.dylib (0x22572a688) and /Applications/Tabby.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Libraries/libGLESv2.dylib (0x1077153c8). One of the two will be used. Which one is undefined.
Error returned: <dictionary: 0x1e9103480> { count = 1, transaction: 0, voucher = 0x0, contents =
	"XPCErrorDescription" => <string: 0x1e9103618> { length = 18, contents = "Connection invalid" }
}
```

**Comment by jw-12138** [comment]
same here, m1 chip

**Comment by ispiropoulos** [comment]
If it's any helpful, I've built `Tabby` locally and on `yarn start` I am getting this:

```
➜  tabby git:(master) ✗ yarn start
yarn run v1.22.17
$ cross-env TABBY_DEV=1 electron app --debug --inspect
[97202:0609/175241.790119:ERROR:node_bindings.cc(242)] Error parsing Node.js cli flags 
Debugger listening on ws://127.0.0.1:9229/96c84839-39d4-40df-966f-27d2a0ea0c6e
For help, see: https://nodejs.org/en/docs/inspector
App threw an error during load
Error: Invalid argument not valid semver ('' received)
    at validateAndParse (/Users/justauser/Projects/Personal/tabby/app/dist/webpack:/tabby/node_modules/compare-versions/index.mjs:65:11)
    at compareVersions (/Users/justauser/Projects/Personal/tabby/app/dist/webpack:/tabby/node_modules/compare-versions/index.mjs:3:14)
    at compare (/Users/justauser/Projects/Personal/tabby/app/dist/webpack:/tabby/node_modules/compare-versions/index.mjs:32:15)
    at Object.call (/Users/justauser/Projects/Personal/tabby/app/dist/webpack:/tabby/lib/window.ts:29:59)
    at n (/Users/justauser/Projects/Personal/tabby/app/dist/webpack:/tabby/webpack/bootstrap:19:32)
    at Object.call (/Users/justauser/Projects/Personal/tabby/app/dist/webpack:/tabby/lib/app.ts:10:1)
    at n (/Users/justauser/Projects/Personal/tabby/app/dist/webpack:/tabby/webpack/bootstrap:19:32)
    at Object.call (/Users/justauser/Projects/Personal/tabby/app/dist/webpack:/tabby/lib/index.ts:8:1)
    at __webpack_require__ (/Users/justauser/Projects/Personal/tabby/app/dist/webpack:/tabby/webpack/bootstrap:19:32)
    at /Users/justauser/Projects/Personal/tabby/app/dist/webpack:/tabby/webpack/startup:4:27
```

**Comment by ispiropoulos** [comment]
> release a new version plz

macOS 13 is in early beta stage. If you cannot wait, you can pull master and build it yourself following the guide [here](https://github.com/Eugeny/tabby/blob/master/HACKING.md)

Thank you @Eugeny for the "swift" patch!

**Comment by Eugeny** [comment]
The nightly build should be ready soon: https://github.com/Eugeny/tabby/actions/runs/2473422987

**Comment by kshq1996** [comment]
> The nightly build should be ready soon: https://github.com/Eugeny/tabby/actions/runs/2473422987
> 
> > <img alt="" width="48" src="https://repository-images.githubusercontent.com/77213120/49dde400-c8c2-11e9-812a-e1407f1f42af"><img alt="" height="14" src="https://camo.githubusercontent.com/b6a12909f1e31185a69a73d59208c507a992236d3230f9fc18e85058ae3d19e7/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f66617669636f6e732f66617669636f6e2e737667"> GitHub**[bumped macos-release - fixes #6564 - startup on macOS Ventura · Eugeny/tabby@52dd0db](https://github.com/Eugeny/tabby)**A terminal for a more modern age. Contribute to Eugeny/tabby development by creating an account on GitHub.

I got it, it built successfully, but it seems that I can't download it , the total size shows like '?'

**Comment by ispiropoulos** [comment]
> > The nightly build should be ready soon: https://github.com/Eugeny/tabby/actions/runs/2473422987
> > > <img alt="" width="48" src="https://repository-images.githubusercontent.com/77213120/49dde400-c8c2-11e9-812a-e1407f1f42af"><img alt="" height="14" src="https://camo.githubusercontent.com/b6a12909f1e31185a69a73d59208c507a992236d3230f9fc18e85058ae3d19e7/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f66617669636f6e732f66617669636f6e2e737667"> GitHub**[bumped macos-release - fixes #6564 - startup on macOS Ventura · Eugeny/tabby@52dd0db](https://github.com/Eugeny/tabby)**A terminal for a more modern age. Contribute to Eugeny/tabby development by creating an account on GitHub.
> 
> I got it, it built successfully, but it seems that I can't download it , the total size shows like '?'
> 
> > <img alt="" width="48" src="https://repository-images.githubusercontent.com/77213120/49dde400-c8c2-11e9-812a-e1407f1f42af"><img alt="" height="14" src="https://camo.githubusercontent.com/b6a12909f1e31185a69a73d59208c507a992236d3230f9fc18e85058ae3d19e7/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f66617669636f6e732f66617669636f6e2e737667"> GitHub**[bumped macos-release - fixes #6564 - startup on macOS Ventura · Eugeny/tabby@52dd0db](https://github.com/Eugeny/tabby)**A terminal for a more modern age. Contribute to Eugeny/tabby development by creating an account on GitHub.

@kshq1996 here:
https://nightly.link/Eugeny/tabby/workflows/build/master/macOS%20.zip%20%28arm64%29.zip

**Comment by kshq1996** [comment]
> > > The nightly build should be ready soon: https://github.com/Eugeny/tabby/actions/runs/2473422987
> > > > <img alt="" width="48" src="https://repository-images.githubusercontent.com/77213120/49dde400-c8c2-11e9-812a-e1407f1f42af"><img alt="" height="14" src="https://camo.githubusercontent.com/b6a12909f1e31185a69a73d59208c507a992236d3230f9fc18e85058ae3d19e7/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f66617669636f6e732f66617669636f6e2e737667"> GitHub**[bumped macos-release - fixes #6564 - startup on macOS Ventura · Eugeny/tabby@52dd0db](https://github.com/Eugeny/tabby)**A terminal for a more modern age. Contribute to Eugeny/tabby development by creating an account on GitHub.
> > 
> > 
> > I got it, it built successfully, but it seems that I can't download it , the total size shows like '?'
> > > <img alt="" width="48" src="https://repository-images.githubusercontent.com/77213120/49dde400-c8c2-11e9-812a-e1407f1f42af"><img alt="" height="14" src="https://camo.githubusercontent.com/b6a12909f1e31185a69a73d59208c507a992236d3230f9fc18e85058ae3d19e7/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f66617669636f6e732f66617669636f6e2e737667"> GitHub**[bumped macos-release - fixes #6564 - startup on macOS Ventura · Eugeny/tabby@52dd0db](https://github.com/Eugeny/tabby)**A terminal for a more modern age. Contribute to Eugeny/tabby development by creating an account on GitHub.
> 
> @kshq1996 here: https://nightly.link/Eugeny/tabby/workflows/build/master/macOS%20.zip%20%28arm64%29.zip

![image](https://user-images.githubusercontent.com/25955989/173040876-474acb39-68e1-4b0c-9860-cfac8e7cead1.png)
total size is ?
